src/dags/bouwstroompunten.py

```diff
@@ -38,34 +41,33 @@
 count_checks = []
 geo_checks = []
 
+db_conn: object = DatabaseEngine()
 
-# data connection
-def get_data():
-    """getting the the access token and calling the data endpoint"""
 
-    # get token
-    token_url = f"{base_url}/{auth_endpoint}"
+def get_token() -> Any:
```

Comment:
By returning `Any` you are basically turning type annotations off. If possible you should be more specific.

You seem to be doing `return token_load["jwt"]`. What is that. Is it:
* a Base64Url encode string? If so, annotated the function return type with `-> str`
* a decoded JWT token? If so, annotate the function return type with `-> Dict[str, str] (assuming it's a dict, don't know much about JWT)

Reply:
I suspect is should be `-> str` not that I have seen how it is used in:

```python
...
"Authorization": f"Bearer {get_token()}",
...
```